node/mock-answer.ts

```diff
@@ -15,10 +15,25 @@ export interface TaskLibAnswers {
     find?: { [key: string]: string[] },
     findMatch?: { [key: string]: string[] },
     ls?: { [key: string]: string },
+    osType?: { [key: string]: string },
     rmRF?: { [key: string]: { success: boolean } },
-    which?: { [key: string]: string; },
+    stats?: { [key: string]: any }, // Can't use `fs.Stats` as most existing uses don't mock all required properties
+    which?: { [key: string]: string },
 }
 
+// TODO TypeScript 2.1: use `keyof`
```

Comment:
What does this mean?

Reply:
I believe we're using version 1.8.7 of the TypeScript compiler.  TypeScript 2.1 adds a `keyof` operator: you can just say `keyof TaskLibAnswer` instead of defining `MockedCommand`.